### ISSUE TITLE
Failed to load the gitignore-snippets package

### ISSUE THREAD

**Issue by adam-stokes** [jefkoslowski/gitignore-snippets#3]
[Enter steps to reproduce below:]
1. atom -1

**Atom Version**: 0.198.0 :warning: **in 1.0 API Preview Mode** :warning:
**System**: linux 3.19.0-16-generic
**Thrown From**: [gitignore-snippets](https://github.com/jefkoslowski/gitignore-snippets) package, v0.2.1
### Stack Trace

Failed to load the gitignore-snippets package

```
At Cannot read property 'prototype' of undefined

TypeError: Cannot read property 'prototype' of undefined
    at __extends (/home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets-view.coffee:4:199)
    at /home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets-view.coffee:9:5
    at Object.<anonymous> (/home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets-view.coffee:52:5)
    at Object.<anonymous> (/home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets-view.coffee:54:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
    at Module.require (module.js:357:17)
    at require (module.js:376:17)
    at Object.<anonymous> (/home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets.coffee:4:27)
    at Object.<anonymous> (/home/adam/.atom/packages/gitignore-snippets/lib/gitignore-snippets.coffee:21:4)
    at Module._compile (module.js:452:26)
    at Object.requireCoffeeScript (/usr/share/atom/resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
    at Module.load (module.js:347:32)
    at Function.Module._load (module.js:302:12)
```
### Commands

```
```
### Config

``` json
{
  "core": {
    "autoHideMenuBar": true,
    "disabledPackages": [
      "go-format"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
atom-rails, v0.4.0
autocomplete-css, v0.7.2
autocomplete-html, v0.7.2
autocomplete-paths, v1.0.2
autocomplete-plus, v2.15.1
autocomplete-python, v0.3.0
autocomplete-ruby, v0.1.0
autocomplete-snippets, v1.6.1
color-picker, v1.7.0
file-icons, v1.5.5
gitignore-snippets, v0.2.1
gulp-snippets, v0.6.1
linter, v0.12.2
linter-flake8, v1.4.0
linter-jshint, v0.1.4
linter-pep8, v0.1.0
linter-rubocop, v0.2.3
ruby-block, v0.3.3
travis-ci-status, v0.14.1

# Dev
No dev packages
```

**Comment by adam-stokes** [comment]
Fixed in 0.2.3, thanks!

**Comment by jefkoslowski** [comment]
Awesome, you're welcome.
